code/modules/admin/verbs/shuttlepanel.dm

```diff
@@ -37,7 +37,13 @@
 		if("Delete Shuttle")
 			if(tgui_alert(user, "Really delete [name || shuttle_id]?", "Delete Shuttle", list("Cancel", "Really!")) != "Really!")
 				return
-			jumpToNullSpace()
+			//VOID EDIT [
+			if(istype(src, /obj/docking_port/mobile/voidcrew))
+				var/obj/docking_port/mobile/voidcrew/dockingport = src
+				dockingport.current_ship.destroy_ship(TRUE)
+			else
+				jumpToNullSpace()
+			// ]
```

Comment:
This is from admin deleting, do we need to have these warnings?

Reply:
The latejoin menu expects every overmap ship to have a docking port, so this was added to ensure both of them get deleted. This could be removed but the overmap object probably shouldn't exist after the shuttle doesn't